rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -92,9 +95,12 @@ def test_calculate_spend_cost_basis_1_buy_consumed_by_1_sell(accountant):
     )
 
     spending_amount = FVal(5)
-    cinfo = cost_basis.calculate_spend_cost_basis(
-        spending_amount=spending_amount,
-        spending_asset=asset,
+    cinfo = cost_basis.spend_asset(
```

Comment:
`spend_asset()` does more things than `calculate_spend_cost_basis() used to do. Are you sure replacing it all over is the right test?

Reply:
Yes, it does for fiat assets & non-taxable spends i.e it calls `reduce_asset_amount()`

But for the tests, the arguments passed ensure only `calculate_spend_cost_basis()` is called.

Reply:
yes but why do this? Don't you see the problem of testing like this?

You are using a different function than the one you are trying to to test. Sure it works now.

But the moment a developer in the team tries to refactor things or move code around everything will break.